### ISSUE TITLE
Documents List | Accessibility: Receipt Date and Document Type should be buttons

### ISSUE THREAD

**Issue by abbyraskin** [department-of-veterans-affairs/caseflow#1847]
**Story**

As a user, I want to be able to sort by receipt date and document type by using my keyboard. Current functionality is this: 
![image](https://cloud.githubusercontent.com/assets/3885236/26321857/968f5642-3ef6-11e7-9886-63f7e791114e.png)

## AC

- [ ] Enable Receipt Date and Document Type as buttons
- [ ] Verify that they can be focused via tab on the keyboard
- [ ] Pressing enter on the keyboard while one of these buttons is selected sorts the given column

**Comment by astewarttistatech** [comment]
**FAILED**  @mmlumba 

Cannot be focused in on with 'tab' key from keyboard. See below: 
![buttonsreader](https://user-images.githubusercontent.com/23038505/27341198-8bf8c6da-55aa-11e7-82ad-8ad5d45d4d23.gif)

**Comment by astewarttistatech** [comment]
**PASSED** 

Validated in: UAT
Browsers: FF and Chrome 

Steps to Validate:
1. Access Reader
2. In the Document List view page, tab between the column  names and on "Receipt Date" and "Document Type" select enter to view that they do sort by those categories. 
3. Make sure that the "Receipt Date" and "Document Type" headers are also functional buttons by clicking on them as well. 

Screenshots: 
![readertabbing](https://user-images.githubusercontent.com/23038505/27386559-3cb4c17c-5664-11e7-8ef7-1b052c15d9fb.gif)
